source/elements/oneTBB/source/task_scheduler/task_arena/task_arena_cls.rst

```diff
@@ -202,6 +247,48 @@ Member functions
         Any number of threads outside of the arena can submit work to the arena and be blocked.
         However, only the maximal number of threads specified for the arena can participate in executing the work.
 
+Example
+-------
+
+The example demonstrates task_arena NUMA support extensions. Each constructed task_arena is pinned
+to the corresponding node. If the platform does not have NUMA capabilities, the example will create
+only one ``task_arena`` and threads entering this ``task_arena`` will not be bound to any hardware
+resources.
```

Comment:
I think we don't need to clarify too precisely what is happening here. This is the spec. We only need to tell that this is an example of how work can be executed on different NUMA nodes simultaneously.